Intro-to-SE-Lab-Fall-20/Group-1
# #22 #26 Reply email Merge with master

Added reply functionality.
Passes the replyed email through the props into the email composition component which displays the email below the reply field. In the future, #23 can be implemented in a very similar manner.

Testing:
- Reply to an email that displays HTML.
- Reply to an email that displays only text.
- Send a normal email.
- Check the sent email and make sure it looks OK.


closes #22 
closes #26
